### PR TITLE
Issue/132 sorting table

### PR DESCRIPTION
**Pull request by ericwoude**

### CODE REVIEW

File: src/components/Table/Table.js [hunk 1]
```diff
@@ -50,49 +64,38 @@ class DataTable extends React.Component {
...
-                        })
-                    }
-
-                </tbody>
-            </table>
-      )
+        /* Creates the table header and binds the rows to the data in the columns
+         * using the accessor. Also manually sets width for first row. */
+        const columns = this.state.headers.map(h => ({Header: h, accessor: h}))
+        columns[0] = {maxWidth: 100, ...columns[0]}
```
Comment (ericwoude) [inline src/components/Table/Table.js]: I dont like this at all, but I couldn't find a more convenient way that still uses map.

Comment (emilhaugberg) [reply src/components/Table/Table.js]: why only the first element?

Comment (ericwoude) [reply src/components/Table/Table.js]: First element is always the "Nr" column, but i've changed it now to a function that chooses width based on the length.

**Comment by ericwoude** [comment]
Able to merge now

**Review by emilhaugberg — APPROVED** [review]

**Pull request merged**